.changeset/itchy-poems-shout.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': minor
+'@itwin/itwinui-css': minor
+---
+
+Expandable block animation now uses pure css for transitioning and `WithCSSTransition` has been removed.
```

Comment:
should be a patch change and should not mention internal names like "WithCSSTransition"

```suggestion
---
'@itwin/itwinui-react': patch
'@itwin/itwinui-css': patch
---

Expandable block animation now uses pure css instead of javascript for transitioning.
```